### PR TITLE
Add panel that shows incoming requests from WAG clients

### PR DESCRIPTION
**Pull request by rgarcia**
https://clever.atlassian.net/browse/INFRA-2396

deployed in dev-infra: https://dev-infra--grafana-service.int.clever.com/dashboard/script/scripted_services.js?env=production&app=app-view-service&from=now-24h&to=now

### CODE REVIEW

**Comment by rgarcia** [comment]
cc @nathanleiby @kvigen went with request sum

File: dashboards/scripted_services.js [hunk 1]
```diff
@@ -1303,7 +1332,115 @@ function createAppLifecycleAnnotations(env, app) {
...
+        ],
+        "dsType": "elasticsearch",
+        "bucketAggs": [
+          {
+            "type": "terms",
+            "field": "container_app",
+            "id": "3",
+            "fake": true,
+            "settings": {
+              "order": "desc",
```
Comment (alsmola) [inline dashboards/scripted_services.js]: It's a little weird to me to order the apps in descending rather than ascending order alphabetically.

**Comment by nathanleiby** [comment]
🙇 those are some impressive gif skillz

File: dashboards/scripted_services.js (hunk 1 above)
Comment (rgarcia) [inline dashboards/scripted_services.js]: The "desc" here is a little misleading--the bar in the graph is stacked ascending alphabetically:

image

^ apps-dashboard is the top sliver, followed by connections-to-s3, etc. The tooltip is somewhat confusingly in descending order, though. Can't win?

**Pull request merged**